bindings/node/src/index.ts

```diff
@@ -0,0 +1,76 @@
+import * as bindings from 'bindings';
+
+const mc = bindings('mongocrypt');
+
+export interface MongoCryptCtor {
```

Comment:
I tried this but we end up with "duplicate identifier" errors, because we also export an object named `MongoCrypt`.  I thought the current approach was better than renaming one or the other.

Reply:
Yea I used `declare class MongoCryptClass` like my suggestion to avoid that, it just allows us to use static keyword, constructor keyword, other familiar syntax.  But this approach is fine if we can make the naming updates.